### ISSUE TITLE
Improve caching of teacher ensembles

### ISSUE THREAD

**Issue by eu9ene** [mozilla/firefox-translations-training#544]
Currently, the name of the teacher training task includes the number of models in the ensemble:
```
"{src_locale}-{trg_locale}-{{this_chunk}}/{{total_chunks}}"
```

Our usual use case is to experiment with one teacher and if we have good results to run the whole pipeline. This means `total_chunks` will be switched from 1 to 2. Ideally, we want not to retrain one of the models and use the cached one instead.

The fix seems to be removing `total_chunks` from the name. Maybe bere there's something else. cc @bhearsum

**Comment by bhearsum** [comment]
That seems sensible to me, yeah. The only reason I put the total chunks in the name was because it's what we do in chunked jobs on other projects.

If you care about having `total_chunks` in the name you could also fiddle with it in https://github.com/mozilla/firefox-translations-training/blob/b7e0a2732c45781626f2f7f7b291edae3e4e97e4/taskcluster/translations_taskgraph/transforms/cached_tasks.py#L98, although that might be a bit more hacky, and you're going to bust existing caches either way.

**Comment by gregtatum** [comment]
Resolved by #606.
